R/magora_corrplot.R

```diff
@@ -0,0 +1,28 @@
+#' Correlation plot for nanostring versus AMPAD data
+#'
+#' Creates a correlation plot comparing nanostring data to human genetic data, faceted by age group and cluster.
+#'
+#' @param data Input data (\code{\link{nanostring_for_plot}}
+#'
+#' @return A ggplot2 object.
+magora_corrplot <- function(data) {
+
+  ggplot2::ggplot() +
+    ggplot2::geom_tile(data = data, ggplot2::aes(x = .data$module, y = .data$model_sex), colour = "black", fill = "white") +
+    ggplot2::geom_point(data = dplyr::filter(data, .data$significant), ggplot2::aes(x = .data$module, y = .data$model_sex, colour = .data$estimate, size = abs(.data$estimate))) +
+    ggplot2::scale_x_discrete(position = "top") +
+    ggplot2::scale_size(guide = "none", limits = c(0, 0.5)) +
+    ggplot2::scale_color_gradient2(limits = c(-0.5, 0.5), breaks = c(-0.5, 0, 0.5), low = "#2166AC", high = "#B2182B", name = "Correlation", guide = ggplot2::guide_colorbar(ticks = FALSE)) +
```

Comment:
Do you think we could use some of the sage colors here? Maybe "blueberry" (`#164B6E`) and "apple" (`#85070C`)?

Reply:
Ooh yes, done! Looks much better, thanks!